phpcs.xml.dist

```
@@ -74,4 +74,7 @@
 			</property>
 		</properties>
 	</rule>
+	<rule ref="WordPress.PHP.NoSilencedErrors.Discouraged">
```

How many times are we silencing errors or how many times are we using `phpcs:ignore` for this rule?

---

I believe it was about 17 times in total

---

This is being reported because the Sniff was renamed 
from `Generic.PHP.NoSilencedErrors.Discouraged` to `WordPress.PHP.NoSilencedErrors.Discouraged`

I'd rather see us change the name in our `phpcs:ignore` statements.

---

Thanks for pointing this out again. I didn't pay attention to this comment in the issue . Sorry :(. Will update the PR.

---

No problem :)